File: models/objects/payments.js
var mongoose = require('mongoose'),
	Schema = mongoose.Schema;
	
var payments_schema = new Schema({
	success: Boolean,
	twitter_response: {type: Schema.Types.Mixed, private: true},
	created_at: {type: Date, default: Date.now},
	account_id: String,
	parrot_id: String,
	message_id: String,
	message_id_sqs: String,
	callback_url: String
});

mongoose.model('Payments', payments_schema);
var payments = mongoose.model('Payments');
module.exports = payments;

// payments.prototype.returnJSON = returnJSON;

File: dal/payparrot_dal/payments.py
# -*- coding: utf-8 -*-
from datetime import datetime
from bson.objectid import ObjectId

from payparrot_dal.base import BaseModel

class Payments(BaseModel):
    _meta = {
        'collection': 'payments',
        'fields': {
            'success': {'required': True},
            'twitter_response': {'private': True},
            'account_id': {'type': ObjectId},
            'action_date': {},
            'created_at': {'default': datetime.now, 'readonly': True, 'private': True},
            'parrot_id': {'type': ObjectId},
            'message_id': {'type': ObjectId},
            'message_id_sqs': {},
            'callback_url': {}
        }
    }